src/Symfony/Component/Messenger/Tests/Transport/RedisExt/ConnectionTest.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Tests\Transport\RedisExt;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Messenger\Exception\LogicException;
+use Symfony\Component\Messenger\Transport\RedisExt\Connection;
+
+/**
+ * @requires extension redis
+ */
+class ConnectionTest extends TestCase
```

Comment:
Spelling in methods `testFromInvalidDns` `testFromDns` `testFromDnsWithOptions`. Must be `Dsn`?

Reply:
@onEXHovia thx fixed!